README.md

```diff
@@ -1,31 +1,37 @@
-# JekyllPagesApi [![Build Status](https://travis-ci.org/18F/jekyll_pages_api.svg?branch=master)](https://travis-ci.org/18F/jekyll_pages_api)
+# Jekyll Pages API [![Build Status](https://travis-ci.org/18F/jekyll_pages_api.svg?branch=master)](https://travis-ci.org/18F/jekyll_pages_api)
 
-TODO: Write a gem description
+Jekyll Pages API is a [Jekyll](http://jekyllrb.com) [Plugin](http://jekyllrb.com/docs/plugins/) that generates a JSON file with data for all the Pages in your Site.
 
-## Installation
+## Usage
 
 Add this line to your application's Gemfile:
 
 ```ruby
-gem 'jekyll_pages_api'
+group :jekyll_plugins do
+  gem 'jekyll_pages_api', git: 'https://github.com/18F/jekyll_pages_api.git'
+end
 ```
 
 And then execute:
 
-    $ bundle
-
-Or install it yourself as:
-
-    $ gem install jekyll_pages_api
-
-## Usage
-
-TODO: Write usage instructions here
+```bash
+bundle
+bundle exec jekyll serve
+```
 
-## Contributing
+You can then see the generated JSON file at http://localhost:4000/api/v1/pages.json, which will look something like this:
+
+```json
+{
+  "entries": [
+    {
+      "title": "18F Hub",
+      "url": "/",
+      "body": "18F is a digital services team within GSA..."
+    },
+    ...
+  ]
+}
+```
 
-1. Fork it ( https://github.com/[my-github-username]/jekyll_pages_api/fork )
```

Comment:
To the first point: Not strongly, no. To the second: Guess we'll see what they say about 18F/open-ource-policy#32.